coss/opensource_clubs/models.py

```diff
@@ -7,13 +7,21 @@
 from wagtail.wagtailcore.fields import RichTextField
 from wagtail.wagtailcore.models import Orderable, Page
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
-
+from wagtail.wagtailsnippets.edit_handlers import SnippetChooserPanel
 
 class CategoryLandingPage(Page):
     description = RichTextField(blank=True)
+    discourse = models.ForeignKey(
+        'discourse.DiscourseCategoryLatest',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
```

Comment:
Shouldn't this be `CASCADE`? I guess if the category gets deleted, we want to make the same for the this listing too, right?